### DISCUSSION TITLE
Include Cut Point or Chapter Markers in output files

### DISCUSSION THREAD

**Discussion by glenpinn** [mifi/lossless-cut#2199] [Support]
Hi there, when i cut (at keyframes) and join my 4k/60p HEVC video files, the Cut or Chapter markers are not visible when i play the output file in any media player on my laptop, and they are not shown when i import the file into my VRD Pro software either.

Is there a setting that i have missed, as i like to see these cut or chapter markers when playing videos, or importing to other editing software.

Just for reference, VRD Pro does leave these markers embedded in the output file.

Cheers

![1 VRD Cut Markers](https://github.com/user-attachments/assets/65bf1857-5123-4143-89ac-ee296f9ed169)
![2 VRD File in MPC-hc](https://github.com/user-attachments/assets/957079ea-6213-49f7-92f6-1f22b9bf5337)

**Comment by mifi** [comment]
Hi. There's a setting called "Create chapters from merged segments (slow) " in the Export confirmation dialog

**Comment by glenpinn** [comment]
Sorry but i cannot locate "Create chapters from merged segments (slow) " in the Export confirmation dialog, unless i am blind.

Cheers

**Comment by mifi** [comment]
see here:
<img width="813" alt="Screenshot 2024-10-22 at 14 47 46" src="https://github.com/user-attachments/assets/cce2f53f-40b7-4ffe-8d9a-5931a143fdfe">

make sure you set "export mode" to "merge cuts":

**Comment by glenpinn** [comment]
Thank you, silly me, i checked the settings after i imported a file but had not made any cuts, so that option is not shown in the output settings box.

Cheers
